src/main/java/duke/command/EventCommand.java

```diff
@@ -10,17 +10,28 @@
 public class EventCommand extends Command {
 
     private final String description;
-    private final DukeDateTime by;
+    private final DukeDateTime at;
```

Comment:
Very good choice abstracting the concept of DateTime and creating an association class for that!